src/lib/Scenes/Collection/Components/FeaturedArtists.tsx

```diff
@@ -0,0 +1,118 @@
+import { Box, Button, EntityHeader, Flex, Sans } from "@artsy/palette"
+import SwitchBoard from "lib/NativeModules/SwitchBoard"
+import React from "react"
+import { TouchableHighlight, TouchableWithoutFeedback } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+
+interface FeaturedArtistsProps {
+  featuredArtists: any
+}
+
+interface FeaturedArtistsState {
+  showMore: boolean
+}
+
+export class FeaturedArtists extends React.Component<FeaturedArtistsProps, FeaturedArtistsState> {
+  state = {
+    showMore: false,
+  }
+
+  handleTap = (context: any, href: string) => {
+    SwitchBoard.presentNavigationViewController(context, href)
+  }
+
+  getFeaturedArtistEntityCollection = (artists: any) => {
```

Comment:
We try not to use `any` type much, since it introduces a lot of uncertainty into the code. Since Emission code can't be rolled back if there is a bug, it's extra-important to avoid bugs, and our types help us with that 👍 In this case, there are two options. I'll outline them both, and then discuss which one I recommend.

- Since this is `merchandisableArtists ` used in the fragment below, once we change the `featuredArtists` prop to be `FeaturedArtists_featuredArtists`, then we can move from `any` to `FeaturedArtists_featuredArtists["artworksConnection "]["merchandisableArtists"]`. 
- `getFeaturedArtistEntityCollection()` is only used once, so instead of defining this logic in its own function, we can inline inside the `render()` function. This means we don't need to explicitly type `artists` at all, since it'll be inferred from the `featuredArtists` prop.

I recommend the second approach because it's less explicit typing. In general, I recommend a longer `render()` function that doesn't call out to single-purpose functions like this, since the compiler's type inference will handle most things for you. This is especially a good idea for this function, since it doesn't really have any logic in it – we could replace the call to `this.getFeaturedArtistEntityCollection(artists)` with the `artists.map(...` call in this function and get the same behaviour with less code/types.

Let me know what I can clarify here 👍

Reply:
For what it's worth, I like having functions broken out of `render`, because it helps me understand everything that's happening in it in one quick pass.

Reply:
That's totally valid, too! I mainly prefer fewer single-use functions because it's less explicit types to write out. Also, it kind of encourages me to refactor a component into smaller components (if the render function is getting too larger). 

In any case, I really only care strongly about the `any` 👍

Reply:
@ashfurrow is it correct to say that one of these ways is more commonly found in iOS dev than the other??

Reply:
@ashleyjelks That's a good question. I'm not sure it's correct to say generally, or even just within Emission. But after a few years of working with React+TypeScript, it is my recommendation based on the trade-offs (it's not a super-strongly-held opinion, though).

My primary concern is typing things as `any`. So as long as we're avoiding those, I'm happy 👍 This is a pretty subtle discussion, and I'm happy to hop on a call to discuss/pair more – just let me know!